classes/sota.bbclass

```diff
@@ -7,6 +7,17 @@ SOTA_CLIENT_PROV ??= "aktualizr-shared-prov"
 SOTA_DEPLOY_CREDENTIALS ?= "1"
 SOTA_HARDWARE_ID ??= "${MACHINE}"
 
+# Translate old provisioning recipe names into the new versions.
```

Comment:
Oh great! Can it also show an advisory warning "name has changed, please be aware..."?

Reply:
Yes! Just need to figure out how to do it in python... otherwise I can try redoing this bit in bash, since I think `bbwarn` does the trick.

Reply:
Finally got around to doing it. It does print a ton of identical warnings, though, but I can't figure out a good way to avoid that.

Reply:
Haha nice. Duplicate warnings are probably fine.